### DISCUSSION TITLE
Bug: ImportError: cannot import name 'AdminConfig' from 'starlette_admin.auth'

### DISCUSSION THREAD

**Discussion by Avin-Techv** [jowilf/starlette-admin#438] [Q&A]
**The bug**
After doing as per the admin example for adding a  login page cannot import AdminConfig as this is not present in the location

**To Reproduce**
try to import like this
from starlette_admin.auth import AdminConfig
or when using code snippet from https://jowilf.github.io/starlette-admin/tutorial/authentication/  with AdminConfig it throws this error

**Environment (please complete the following information):**
 - starlette-admin==0.12.2
 - fastapi==0.105.0
 - certifi==2023.7.22
 - uvicorn==0.24.0.post1

**Comment by dandaniel5** [comment]
There is no AdminConfig in .auth.py. If you're in a rush like me, you can add it manually.
```
@dataclass
class AdminConfig:
    logo_url: Optional[str] = None
    app_title: Optional[str] = None
```
In auth.py and imports to  __init__.

**Comment by hasansezertasan** [comment]
That is not released yet, you can install this repository as a package to use it.

**Comment by dandaniel5** [comment]
> That is not released yet, you can install this repository as a package to use it.

What do you mean when you say "this"? 0.12.2 or starlette-admin in general? If in general, why is it in the pip? But if 0.12.2, then I don't quite understand what you mean by that.

**Comment by hasansezertasan** [comment]
As far as I remember 'AdminConfig' feature is not in the 0.12.2 version.

**Comment by jowilf** [comment]
Thanks @dandaniel5  and @hasansezertasan  for your help.

This feature was added in #374 and will be released in 0.13.0 (https://github.com/jowilf/starlette-admin/milestone/3). 
But, It's is currently available in preview (starlette-admin==0.13.0rc1)

We should have multiple versions of the documentation with  https://github.com/jimporter/mike to avoid this confusion in the future.

> What do you mean when you say "this"? 0.12.2 or starlette-admin in general? If in general, why is it in the pip? But if 0.12.2, then I don't quite understand what you mean by that.

 I think @hasansezertasan was suggesting to install the main branch directly (https://pip.pypa.io/en/stable/topics/vcs-support/)
